projects/libyaml/libyaml_deconstructor_alt_fuzzer.c

```diff
@@ -26,10 +27,9 @@
 int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
   if (size < 2) return 0;
 
-  int k;
-  int done = 0;
-  int canonical = data[0] & 1;
-  int unicode = data[1] & 1;
+  bool is_done = false;
```

Comment:
My fault: I should have mentioned that `done` is usually an exception to our "adjective clause" boolean naming rule; it's pretty common to see:

```c
while(!done) {
  /* ... */
}
```

Reply:
Sure, will do.